include/mixer.h

```diff
@@ -79,26 +79,29 @@ class MixerChannel {
 	void AddStretched(Bitu len,Bit16s * data);		//Strech block up into needed data
 
 	void FillUp(void);
-	void Enable(bool _yesno);
+	void Enable(bool should_enable);
+	void FlushSamples();
 
-	float          volmain[2];
-	MixerChannel*  next;
-	const char*    name;
-	Bitu           done;           //Timing on how many samples have been done by the mixer
-	bool           enabled;
+	float volmain[2] = {0.0f};
```

Comment:
Binary representation of positive 0 in IEEE is indeed 0x00000000, but there's no requirement on C/C++ to implement always use IEEE representation - it's machine-specific - it' likely used in all modern architectures, but we can't rule out, that somewhere, floating point 0 value has different representation.

For this reason, in this specific line I adise to write:

    float volmain[2] = {0.0f, 0.0f};

Reply:
Thanks! (and agree).